src/integ_test/java/games/strategy/engine/lobby/server/db/DbUserControllerIntegrationTest.java

```diff
@@ -16,17 +16,14 @@
 
 public class DbUserControllerIntegrationTest {
```

Comment:
This test should probably be renamed to `UserControllerIntegrationTest`.

Reply:
Sounds reasonable, added in: ede81c7